crud/src/main/java/com/redhat/lightblue/crud/CRUDFindRequest.java

```diff
@@ -140,12 +140,15 @@ public void toJson(JsonNodeFactory factory, ObjectNode node) {
         if (sort != null) {
             node.set("sort", sort.toJson());
         }
-        if (from != null && to != null) {
-            ArrayNode arr = factory.arrayNode();
-            arr.add(from);
-            arr.add(to);
-            node.set("range", arr);
-        }
+		if (from != null && to != null) {
+			ArrayNode arr = factory.arrayNode();
+			arr.add(from);
+			if (to > 0)
```

Comment:
Won't adding a null node for a negative value make the range [x, null] for [x, -y] ?